src/DssSpell.sol

```diff
@@ -17,53 +17,51 @@
+    // Hash: seth keccak -- "$(wget https://raw.githubusercontent.com/makerdao/community/TODO/governance/votes/Executive%20Vote%20-%20March%2025%2C%202022.md -q -O - 2>/dev/null)"
     string public constant override description =
-        "2022-03-11 MakerDAO Executive Spell | Hash: 0x4eaeba62293ee8c19ccb1dfc7d9e97b5764b7e62fefebc1049f2e8f984d0836f";
+        "2022-03-25 MakerDAO Executive Spell | Hash: TODO";
 
-    // Immunefi Payouts
-    address constant WALLET1 = 0x3C32F2ca11D92a7093d1F237161C1fB692F6a8eA;
-    address constant WALLET2 = 0x2BC5fFc5De1a83a9e4cDDfA138bAEd516D70414b;
+    address public constant TUSD_PREV_IMPL = 0xffc40F39806F1400d8278BfD33823705b5a4c196;
+    address public constant TUSD_NEXT_IMPL = 0xd8D59c59Ab40B880b54C969920E8d9172182Ad7b;
 
-    // Flap
-    address constant MCD_FLAP = 0xa4f79bC4a5612bdDA35904FDF55Fc4Cb53D1BFf6;
-
-    // Math
-    uint256 constant RAD = 10**45;
+    function officeHours() public override returns (bool) {
+        return false;
+    }
 
     function actions() public override {
-        onboardNewCollaterals();
-
-        // Immunefi Payouts
-        // https://forum.makerdao.com/t/bounty-payout-request-for-immunefi-bug-5565/13545
-        DssExecLib.sendPaymentFromSurplusBuffer(WALLET1, 2_500);
-        DssExecLib.sendPaymentFromSurplusBuffer(WALLET2, 250);
+        // onboardNewCollaterals();
 
-        // Replace Flapper with rate limit one
-        // https://vote.makerdao.com/polling/Qmdd4Pg7
-        address MCD_VOW = DssExecLib.vow();
-        address MCD_FLAP_OLD = DssExecLib.flap();
-        DssExecLib.setValue(MCD_FLAP, "beg", FlapAbstract(MCD_FLAP_OLD).beg());
-        DssExecLib.setValue(MCD_FLAP, "ttl", FlapAbstract(MCD_FLAP_OLD).ttl());
-        DssExecLib.setValue(MCD_FLAP, "tau", FlapAbstract(MCD_FLAP_OLD).tau());
-        DssExecLib.setValue(MCD_FLAP, "lid", 150_000 * RAD);
-        DssExecLib.deauthorize(MCD_FLAP_OLD, MCD_VOW);
-        DssExecLib.authorize(MCD_FLAP, MCD_VOW);
-        DssExecLib.setContract(MCD_VOW, "flapper", MCD_FLAP);
+        // Update TUSD implementation
+        // https://forum.makerdao.com/t/2022-03-25-adding-support-for-the-new-tusd-implementation-address/14189
+        address MCD_JOIN_TUSD_A = DssExecLib.getChangelogAddress("MCD_JOIN_TUSD_A");
+        GemJoin6Like(MCD_JOIN_TUSD_A).setImplementation(TUSD_PREV_IMPL, 0);
+        GemJoin6Like(MCD_JOIN_TUSD_A).setImplementation(TUSD_NEXT_IMPL, 1);
 
-        // Changelog updates
-        DssExecLib.setChangelogAddress("MCD_FLAP", MCD_FLAP);
-        DssExecLib.setChangelogVersion("1.11.0");
+        // Authorize ESM on DssFlash
+        // https://forum.makerdao.com/t/proposed-security-fix-to-be-added-in-the-march-25th-2022-executive-spell
+        address MCD_FLASH = DssExecLib.getChangelogAddress("MCD_FLASH");
+        address MCD_ESM = DssExecLib.esm();
+        AuthLike(MCD_FLASH).rely(MCD_ESM);
```

Comment:
Can replace these three lines with:
```
DssExecLib.authorize(DssExecLib.getChangelogAddress("MCD_FLASH"), DssExecLib.esm());
```

And get rid of the interface above.

Reply:
done